scripts/skills/racial/rf_zombie_racial.nut

```diff
@@ -0,0 +1,99 @@
+this.rf_zombie_racial <- ::inherit("scripts/skills/skill", {
+	m = {
+		FatigueEffectMult = 0.0,
+		FatigueDealtPerHitBonusMult = 1.0
```

Comment:
A better name would be `FatigueDealtPerHitMultModifier` showing that it is added to the `FatigueDealtPerHitMult` just like we do with other properties e.g. `MeleeSkillModifier` etc.